includes/forms/class-wp-job-manager-form-submit-job.php

```diff
@@ -40,6 +40,13 @@ class WP_Job_Manager_Form_Submit_Job extends WP_Job_Manager_Form {
 	 */
 	protected $preview_job;
 
+	/**
+	 * Renew listing helper instance.
+	 *
+	 * @var WP_Job_Manager_Form_Submit_Job
+	 */
+	public $renew_listing_helper;
```

Comment:
I don't think that this is needed. We can access the instance from everywhere with `WP_Job_Manager_Helper_Listing_Renewal::instance`. I

Reply:
Ok. Changed in https://github.com/Automattic/WP-Job-Manager/pull/2422/commits/c8c7b1ac8a5ab88988bbc446b185fd4f373dbe41